config/chime_science_run_gpu.yaml

```diff
@@ -91,6 +91,58 @@ pulsar_gain:
     - /mnt/gong/cherry/FRB-GainFiles/broker_CygA_10_26_scl/
     - /mnt/gong/cherry/FRB-GainFiles/broker_CygA_10_26_scl/
 
+pulsar_pointing:
```

Comment:
For generating endpoints automatically from the kotekan config in the km-replacement (https://github.com/chime-experiment/coco/issues/31), the I propose that we stick to a common place for the updatable config blocks (like [here](https://github.com/kotekan/kotekan/blob/556a62159b0a303f652febb1f851751f64b573b9/config/chime_science_run_recv.yaml#L103)). That way, the path in the config is the same as the endpoint name. That way, coco can get the endpoint name from the kotekan config.

Reply:
@nritsche would it not work to just generate the endpoint based on the paths to `kotekan_update_endpoint` keys in the config?   Currently that's how kotekan generates the endpoint paths.   Having said that we did say for standardization reasons we'd put all the updatable config options in an `updatable_config` block.  So I think it's still worth doing that for consistency - but I'm not sure we want that as a hard requirement.

Reply:
Oh yeah you are right. So if this value is always called `kotekan_update_endpoint`, I can use it (this is also not enforced).